pystoned/CNLS.py

```diff
@@ -135,8 +135,7 @@ def regression_rule(model, i):
 
                 return regression_rule
             elif self.rts == RTS_CRS:
-                # TODO(warning handling): replace with model requested not exist
-                return False
+                raise Exception("The Additive CNLS with constant return to scale does not exist")

```

Comment:
Although additive CNLS-CRS models are rarely used in the application, they should be added to our package. So, could you please add CRS specifications to all additive CNLS models?